src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -220,72 +109,49 @@ function WorkspaceInvitePage(props) {
-                        onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
-                    >
-                        <HeaderWithBackButton
-                            title={translate('workspace.invite.invitePeople')}
-                            subtitle={policyName}
-                            shouldShowGetAssistanceButton
-                            guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
-                            onBackButtonPress={() => {
-                                Policy.clearErrors(props.route.params.policyID);
-                                Navigation.goBack(ROUTES.WORKSPACE_MEMBERS.getRoute(props.route.params.policyID));
-                            }}
-                        />
-                        <SelectionList
-                            canSelectMultiple
-                            sections={sections}
-                            textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
-                            textInputValue={searchTerm}
-                            onChangeText={(value) => {
-                                SearchInputManager.searchInput = value;
-                                setSearchTerm(value);
-                            }}
-                            headerMessage={headerMessage}
-                            onSelectRow={toggleOption}
-                            onConfirm={inviteUser}
-                            showScrollIndicator
-                            showLoadingPlaceholder={!didScreenTransitionEnd || !OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
-                            shouldPreventDefaultFocusOnSelectRow={!Browser.isMobile()}
+            {({didScreenTransitionEnd}) => (
+                <FullPageNotFoundView
+                    shouldShow={(_.isEmpty(props.policy) && !props.isLoadingReportData) || !PolicyUtils.isPolicyAdmin(props.policy) || PolicyUtils.isPendingDeletePolicy(props.policy)}
+                    subtitleKey={_.isEmpty(props.policy) ? undefined : 'workspace.common.notAuthorized'}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
+                >
+                    <HeaderWithBackButton
+                        title={translate('workspace.invite.invitePeople')}
+                        subtitle={policyName}
+                        shouldShowGetAssistanceButton
+                        guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
+                        onBackButtonPress={() => {
+                            Policy.clearErrors(props.route.params.policyID);
+                            Navigation.goBack(ROUTES.WORKSPACE_MEMBERS.getRoute(props.route.params.policyID));
+                        }}
+                    />
+                    <MemberInviteList
+                        didScreenTransitionEnd={didScreenTransitionEnd}
+                        inviteUsers={inviteUser}
```

Comment:
@tienifr 

https://github.com/tienifr/App/pull/34

I renamed `inviteUser` to `inviteUsers` to keep it consistent.